stumpy/cache.py

```diff
@@ -220,13 +248,25 @@ def save():
     Returns
     -------
     None
+
+    Notes
+    -----
+    The cache is never cleared before saving/overwriting and may be explicitly
+    cleared by calling `cache.clear()` before saving.
     """
+    global CACHE_CLEARED
```

Comment:
We can remove it if we are not using it here. I can see that this is used in the child function `cache._save`